depinject/provider_desc_test.go

```diff
@@ -45,7 +45,7 @@ func StructInAndOut(_ float32, _ StructIn, _ byte) (int16, StructOut, int32, err
 	return int16(0), StructOut{}, int32(0), nil
 }
 
-func BadErrorPosition() (error, int) { return nil, 0 }
+func BadErrorPosition() (error, int) { return nil, 0 } //nolint:revive,stylecheck // error should be the last type but this seems a test for bad error position
```

Comment:
what is the reason this still has to exist? please dont copy the error but provide reasoning why its not being fixed